### PR TITLE
fix tables, and display overview pages

### PR DESCRIPTION
**Pull request by gunjan5**
- render proper tables
- show overview page for each layer
- fix some Liquid errors

### CODE REVIEW

**Comment by gunjan5** [comment]
temporary preview available at http://16a1a9a6.ngrok.io/master/reference/calicoctl/

File: _data/master/navbars/reference.yml [hunk 1]
```diff
@@ -10,11 +10,15 @@ toc:
 - title: calicoctl
   path: /reference/calicoctl
```
Comment (robbrockbank) [inline _data/master/navbars/reference.yml]: can remove this path.

File: _data/master/navbars/reference.yml [hunk 2]
```diff
@@ -26,6 +30,8 @@ toc:
     - title: node
       path: /reference/calicoctl/commands/node
```
Comment (robbrockbank) [inline _data/master/navbars/reference.yml]: Can remove this path

**Comment by robbrockbank** [comment]
Need to remove old links for the expandable headings:  calicoctl / node / ipam

Looks like semaphore is failing due to link checking - but it's fine to merge once you've removed those headings.   We should open another PR to fix up broken links - if you are short for work that would be good to knock off.

File: _data/master/navbars/reference.yml (hunk 2 above)
Comment (gunjan5) [inline _data/master/navbars/reference.yml]: fixed!

**Comment by gunjan5** [comment]
I'll merge this to `golang` branch, and will do another PR for the broken links

**Pull request merged**